js/whitebit.js

```diff
@@ -974,64 +974,65 @@ module.exports = class whitebit extends Exchange {
          * @param {object} params extra parameters specific to the whitebit api endpoint
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
-        let method = undefined;
         await this.loadMarkets ();
         const market = this.market (symbol);
         const request = {
             'market': market['id'],
             'side': side,
             'amount': this.amountToPrecision (symbol, amount),
         };
-        const stopPrice = this.safeNumber2 (params, 'stopPrice', 'activationPrice');
-        if (stopPrice !== undefined) {
-            // it's a stop order
+        const isLimitOrder = (type === 'limit');
```

Comment:
I believe @frosty00 told me said we will no longer be supporting them.

Reply:
@DoctorSlimm I asked @kroitor, and he also thinks we should support both because it might be beneficial to ccxt; the easier it is to place an order, the more volume ccxt gets, and in this case, it is pretty simple. Can you make that change pls?

Reply:
@carlosmiei sure thing, will do